corehq/apps/style/templates/style/bootstrap3/base.html

```diff
@@ -68,6 +68,16 @@
         {% endcompress %}
         {% endif %}
 
+        {% if request.use_jquery_ui_multiselect %}
+        {% compress css %}
+            <link type="text/css"
+                  rel="stylesheet"
+                  media="screen"
+                  href="{% new_static "hqwebapp/js/lib/jquery-ui/multiselect/ui.multiselect.css" %}" />
```

Comment:
@kkrampa 

Kamil, I've added a new section to the Migration Guide that explains how to add a new javascript library to Bower.js and how to create a decorator for it to use in your view. Sounds like you already got the decorator part down, but just make sure that the css is also carried over. Thank you!

https://docs.google.com/document/d/1jOIrCZlrmECfzo-iyAYjeVFqBv7ilnXjdgDWjGLmqd4/edit#heading=h.ocbqhhpvp8si